src/ad-hoc-visualizations/issues/visualization.tsx

```diff
@@ -33,7 +33,10 @@ export const IssuesAdHocVisualization: VisualizationConfiguration = {
         <AdhocIssuesTestView instancesSection={FailedInstancesSection} {...props} />
     ),
     getStoreData: data => data.adhoc.issues,
-    enableTest: (data, _) => (data.enabled = true),
+    enableTest: (data, _) => {
+        data.adhoc.issues.enabled = true;
+        data.adhoc[AdHocTestkeys.NeedsReview].enabled = false;
```

Comment:
Just out of curiosity, why couldn't you use `data.adhoc.needsReview.enabled = false`? Is there a specific reason you need to specify `[AdHocTestkeys.NeedsReview]`

Reply:
Changed it so it's always using the enums directly.  This way if the enum ever changes, these will still work